sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Options.cs

```diff
@@ -29,8 +29,8 @@ public class Options
         [Option('s', "clientSecret", Required = false, HelpText = "Application client secret. Only applicable when using LoginMethod of AppId.")]
         public string ClientSecret { get; set; }
 
-        [Option('e', "eventHubName", Required = true, HelpText = "Event Hub Name linked to digital twins instance")]
-        public string EventHubName { get; set; }
+        [Option('e', "eventHubEndpointName", Required = true, HelpText = "Event Hub endpoint linked to digital twins instance")]
+        public string EventHubEndpointName { get; set; }
```

Comment:
This is actually the event hub endpoint name that has been linked to the adt instance.